qutebrowser/config/config.py

```diff
@@ -286,6 +288,40 @@ def _transform_position(val):
         return val
 
 
+def _transform_hint_color(val):
+    """Transformer for hint colors."""
+    log.config.debug("Transforming hint value {}".format(val))
+
+    def to_rgba(qcolor):
+        """Convert a QColor to a rgba() value."""
+        return 'rgba({}, {}, {}, 0.8)'.format(qcolor.red(), qcolor.green(),
+                                              qcolor.blue())
+
+    if val.startswith('-webkit-gradient'):
+        pattern = re.compile(r'-webkit-gradient\(linear, left top, '
+                             r'left bottom, '
+                             r'color-stop\(0%, *(#[a-fA-F0-9]{3,6})\), '
+                             r'color-stop\(100%, *(#[a-fA-F0-9]{3,6})\)')
+
+        match = pattern.match(val)
+        if match:
+            log.config.debug('Color groups: {}'.format(match.groups()))
+            start_color = QColor(match.group(1))
+            stop_color = QColor(match.group(2))
+            qtutils.ensure_valid(start_color)
+            qtutils.ensure_valid(stop_color)
+
+            return ('qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 {}, '
```

Comment:
Not that it matters much, but this conversation is not 100% correct. It changes from a gradient top-bottom to a gradient topleft-bottomright, which means it now as a 45° angle compared to before. The correct parameter would be `x2:0`, so that the interpolation line goes from `(0, 0)` to `(0, 1)`, which is top-bottom without angle.

You can barely see it with standard settings, but changing the gradient to black-white and the size to 50px should show you the difference.